aten/src/ATen/core/BFloat16.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
+#include "c10/BFloat16.h"
```

Comment:
You don't really need this file, you can just include c10/BFloat16.h directly

Reply:
👍